src/libcore/intrinsics.rs

```diff
@@ -44,6 +44,10 @@
 
 use marker::Sized;
 
+#[cfg(stage0)] pub use self::copy_memory as copy;
```

Comment:
Can we have some way of marking that lines should be removed with the new snapshot? The amount of permanently-`#[cfg(stage0)]`d code is probably going to keep increasing, and the snapshot maker has to go through this each time.